.circleci/config.yml

```diff
@@ -118,6 +121,16 @@ workflows:
     jobs:
       - build
 
+      - architect/push-to-app-catalog:
+          name: "package and push kubernetes-metrics-server chart"
+          app_catalog: "giantswarm-catalog"
+          app_catalog_test: "giantswarm-test-catalog"
+          chart: "kubernetes-metrics-server-chart"
```

Comment:
Why does it have `-chart` suffix?

Reply:
@kopiczko It's pointing this directory, otherwise I need to create another directory named it differently. https://github.com/giantswarm/kubernetes-metrics-server/tree/master/helm/kubernetes-metrics-server-chart

Reply:
Do we need to have it build by architect at the same time? Otherwise you can just move.

Reply:
> Do we need to have it build by architect at the same time? 

@kopiczko Good point, but what if we need to deploy hot-fix version for an older cluster? Don't we have those cases before?